www/source/demo/windows/steps/4.html.slim

```diff
@@ -0,0 +1,33 @@
+section
+ h1 Define Build Logic For Your App
+
+ p When building your package, your project source (or pre-built binary) goes through a series of steps as part of the package compilation process. Depending on whether or not your Habitat plan resides in the same repo as your project, these steps could include downloading your source, unpacking it, building it, and then installing it into the appropriate location within a Habitat package.
+
+ p The default behavior for these build steps are defined in build phase callbacks, which are functions called by Habitat when building your package. When overriding any #{link_to 'callbacks','/docs/reference#reference-callbacks'}, you may use any of the #{link_to 'plan variables','/docs/reference#plan-variables'}, #{link_to 'settings','/docs/reference#plan-settings'}, or #{link_to 'functions','/docs/reference#utility-functions'} in the #{link_to 'Plan syntax guide', '/docs/reference'}, except for the #{link_to 'runtime configuration settings','/docs/reference#template-data/'}. Those can only be used in runtime hooks once a Habitat service is running.
```

Comment:
I'd suggest linking the first instance of `callbacks` (in the first sentence here) rather than the one in the second.